sgkit/io/vcf/vcf_reader.py

```diff
@@ -261,6 +262,9 @@ def vcf_to_zarrs(
         If True, genotype calls with more alleles than the specified (maximum) ploidy value
         will be truncated to size ploidy. If false, calls with more alleles than the
         specified ploidy will raise an exception.
+    alt_number
```

Comment:
Hmm, not sure ``alt_number`` captures this. What about ``max_alt_alleles``?

Reply:
I was using the scikit-allel term, but happy to go with either.

Reply:
We should probably get more opinions

Reply:
I prefer `max_alt_alleles`.